source/common/upstream/upstream_impl.cc

```diff
@@ -216,7 +233,23 @@ ClusterImplBase::ClusterImplBase(const envoy::api::v2::Cluster& cluster,
                                  Runtime::Loader& runtime, Stats::Store& stats,
                                  Ssl::ContextManager& ssl_context_manager, bool added_via_api)
     : runtime_(runtime), info_(new ClusterInfoImpl(cluster, source_address, runtime, stats,
-                                                   ssl_context_manager, added_via_api)) {}
+                                                   ssl_context_manager, added_via_api)) {
+  priority_set_.addMemberUpdateCb([this](uint32_t, const std::vector<HostSharedPtr>& hosts_added,
+                                         const std::vector<HostSharedPtr>& hosts_removed) {
+    if (!hosts_added.empty() || !hosts_removed.empty()) {
+      info_->stats().membership_change_.inc();
+    }
+
+    uint32_t healthy_hosts = 0;
```

Comment:
maybe add a TODO here for per-priority stats?

Reply:
Sanity check: do you believe anyone wants them yet?
on more thought, I think we can get away with remotely reported load stats and I'd rather not add them unless someone wants them.

Reply:
I can't imagine that anyone that actually uses priority wouldn't want broken out stats, but given that we won't be using this at Lyft right now we can wait until someone requests. :)